lib/AppConfig.php

```diff
@@ -55,7 +56,17 @@ public function getAppNamespace($key) {
 		}
 		return Application::APPNAME;
 	}
-
+	/*
+	*Get config values from main server config and merge two array's 
+	*into one returning array
+	*/
+	public function getConfigValues(){
+		$config = \OC::$server->get(IConfig::class);
```

Comment:
This is already injected through dependency injection in the constructor so no need to query the server container again.